submultimi/submultimi.cpp

```diff
@@ -0,0 +1,31 @@
+//https://infoarena.ro/problema/submultimi
+
+#include <fstream>
+#include <vector>
+
+using namespace std;
+ifstream fin ("submultimi.in");
+ofstream fout ("submultimi.out");
+
+vector <int> submultime;
+
+void solution(int position, int N, int lastValue) {
+    for (int i = lastValue; i <= N; ++i) {
+        submultime.push_back(i);
+        for (int j = 0; j < submultime.size(); ++j) {
+            fout << submultime[j] << " ";
+        }
```

Comment:
This for is ok to be here, but it could have been at the beginning of the method as well to keep the classic backtracking structure.